repaygithub/ui-tools
# CACTUS-381 :: Upgrade out-of-date dependencies in generated app

Ticket: https://repayonline.atlassian.net/browse/CACTUS-381

### Testing

1. Pull down the branch and `cd` into `modules/create-repay-ui`.
2. Run `yalc publish`.
3. Run `yarn global add ~/.yalc/packages/@repay/create-ui/0.2.0`
4. Go to some test directory and run `create-repay-ui`.
5. Generate a JavaScript app and `cd` into it.
6. Run `yarn upgrade-interactive --latest --registry https://registry.yarnpkg.com`.  Verify that it says you're up-to-date.
7. Repeat for a TypeScript app.  Note that it will say that the TypeScript eslint parser is out-of-date.  That's okay, as we're going to fix that in https://repayonline.atlassian.net/browse/CACTUS-323
